Documentation/network/kubernetes/kubeproxy-free.rst

```diff
@@ -506,6 +507,56 @@ enabled would look as follows:
         --set k8sServiceHost=${API_SERVER_IP} \\
         --set k8sServicePort=${API_SERVER_PORT}
 
+.. _DSR mode with Geneve:
+
+Direct Server Return (DSR) with Geneve
+**************************************
+By default, Cilium with DSR mode encodes the service IP/port in a Cilium-specific
+IPv4 option or IPv6 Destination Option extension so that the backends make aware of
+the service IP/port, which they need to reply with.
+
+However, some data center routers pass packets with unknown IP options to software
+processing called "Layer 2 slow path". As well as taking speeds from millions of
+packets, those routers drop the packets if the traffic of packets with IP options
+exceeds the threshold. That significantly affects network performance.
```

Comment:
I find this paragraph hard to read. In particular, I'm not sure what you mean with “As well as taking speeds from millions of packets”. Do you mean these routers are able to process millions of packets per second? Is this relevant in the explanation?

Some suggestions:

```suggestion
However, some data center routers pass packets with unknown IP options to software
processing called "Layer 2 slow path".
Those routers drop the packets if the amount of packets with IP options
exceeds a given threshold, which may significantly affect network performance.
```

Reply:
I just tries to express that the router's soft process is slow, but it's confusing. I'll remove it.